src/devices/sound/flt_biquad.h

```diff
@@ -0,0 +1,256 @@
+// license:BSD-3-Clause
+// copyright-holders:K.Wilkins,Couriersud,Derrick Renaud,Frank Palazzolo,Jonathan Gevaryahu
+#pragma once
+
+#ifndef MAME_SOUND_FLT_BIQUAD_H
+#define MAME_SOUND_FLT_BIQUAD_H
+
+#pragma once
+
+// we need the M_SQRT2 constant
+#ifndef M_SQRT2
+#define M_SQRT2 1.41421356237309504880
+#endif
```

Comment:
Fixed, the constant is now defined only in the .cpp file, and written out in full in the .h. When I get to moving the inline functions from the .h to the .cpp file, this will be less of an issue there as well.